src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -406,7 +407,7 @@ public async Task AddNode()
                         }
                     }
 
-                    var nodeTag = nodeInfo.NodeTag == RachisConsensus.InitialTag ? null : nodeInfo.NodeTag;
+                    var nodeTag = nodeInfo.NodeTag == RachisConsensus.InitialTag ? tag : nodeInfo.NodeTag;
```

Comment:
Do we need to validate if node with given tag already exist in the cluster? @karmeli87 ?

Reply:
I would also suggest applying `.Trim()` on this

Reply:
We do it here:
https://github.com/ravendb/ravendb/blob/cda4226f616ec7dcaac6ab173be79b75c9683134/src/Raven.Server/Rachis/Leader.cs#L905

Reply:
@karmeli87 I tested and there is no issue here, it remembering the bigger tag as the last node tag.

Reply:
@ppekrol In case of adding a node on existing node tag, it reports that the node was added successfully, but the actual case was no-op. I'll fix it to throw an error.